### DISCUSSION TITLE
Vulnerabilities Report

### DISCUSSION THREAD

**Discussion by prabu-sivakumar** [MscrmTools/XrmToolBox#1196] [Q&A]
Good day to you. We are using Microsoft Dynamics 365 CRM platform for managing CRM requirements for our clients in the Bank. To manage deployments, we used solution component mover plugin within the XrmToolbox. In order to onboard the software, we'd require security vulnerabilities to be reviewed by the security architects. If you could help us with the vulnerabilities report, it would be a great help.

**Comment by MscrmTools** [comment]
Hi, 
XrmToolBox is open source so you can find all and everything that XrmToolBox does in its source code.
I can't see how we can help more.

**Reply by prabu-sivakumar** [reply]
Firstly, thanks for your response. Typically when we develop software, we run App Scan on the source code to identify security vulnerabilities. Aqua Scan and IBM App Scan reports are a few examples. If you could share the list of known vulnerabilities (if any), it would be a great help.

**Reply by MscrmTools** [reply]
So, I guess you can do it yourself as the source code is freely available.

**Reply by prabu-sivakumar** [reply]
Yes, we've generated a few vulnerabilities reports where we don't see any significant vulnerabilities reported. Most of the Microsoft products manage the list of known vulnerabilities and their severity. I was wondering if XrmToolbox has one.

**Reply by MscrmTools** [reply]
No, there is not.
